### ISSUE TITLE
updating, how find settings, config etc

### ISSUE THREAD

**Issue by astro46** [tablacus/TablacusExplorerAddons#373]
I find that there is a v24.  I am currently using v. 22.   
Downloaded, unzipped to folder.  moved folder to folder where I had v22 running from. (not program folder)

how do I tell v 24 what addons are installed, configuration, etc.   
I tried copying the v22 'config' folder to the v24 folder.  didn't help. 

Or am I supposed to just copy TE64.exe  from the v24 folder to the v22 folder.  Everything else hasn't changed?

I don't see anything about this in the readme.

thanks,
jeffrey

**Comment by ccfs** [comment]
To update TE properly:
1. Ensure you have enabled "Add-ons updater" add-on.
2. Choose "Help > Check for updates" and reply OK.
3. If you are using TE WebView2 extension, you need to use "WebView2 (Edge
Blink engine)" add-on to update the extension.

Copy your v22 to another folder and try the above steps there.

Alternatively I have been developing a [patched version](https://github.com/tablacus/TablacusExplorer/issues/692) of TE 23.9.13. You can download that version and install the patch. The above steps do not apply to the patch version.

**Comment by astro46** [comment]
that was easy.  I have addons updater, and it is enabled.  I used help>check for updates and it was done. I don't remember why I made it so complicated before. I guess I was trying to download the new version. Perhaps I went throught Help>about>check for updates or visit website and downloaded v24 from there but couldn't then figure out how to integrate it into the v22 that I was running.

I guess all is good now.  
What is the tablacus webview version about?  I have a couple of apps (eg: zoom) that force me to use webview, but I don't see the advantage in it.  Is there an advantage to using webview for tablacus?
